src/DIRAC/RequestManagementSystem/private/RequestValidator.py

```diff
@@ -276,20 +271,31 @@ def setAndCheckRequestOwner(request, remoteCredentials):
         :returns: True if everything is fine, False otherwise
         """
 
-        credDN = remoteCredentials["DN"]
+        credUserName = remoteCredentials["username"]
         credGroup = remoteCredentials["group"]
         credProperties = remoteCredentials["properties"]
 
+        # FIXME: code for backward compatibility with requests created by 8.0 clients
+        if hasattr(request, "OwnerDN"):  # 8.1 requests won't have this set
+            ownershipCheck = hasattr(request, "Owner") or request.OwnerDN
+        else:
+            ownershipCheck = request.Owner
+        # ##
+
         # If the owner or the group was not set, we use the one of the credentials
-        if not request.OwnerDN or not request.OwnerGroup:
-            request.OwnerDN = credDN
+        if not ownershipCheck or not request.OwnerGroup:
+            # FIXME: code for backward compatibility with requests created by 8.0 clients
+            if hasattr(request, "OwnerDN"):
+                request.OwnerDN = remoteCredentials["DN"]
```

Comment:
I am not sure I understand why do you need to set the `OwnerDN` if it has that attribute ?

Reply:
It adds if it is empty. 
I am not sure if it is even ever possible, but this is what is done in v8.0.

Reply:
Yes of course, this is to set the ownerDN from the credentials (if not set) on the server side when uploading the request.  So here, you want to set just the `Owner` I guess.

Reply:
Well I can just simply remove the 3 lines above then...?

Reply:
After applying this PR as patch to the DIRAC certification setup, and having run some integration tests, I am somewhat confident that the changes in this module cover our cases.